addons/point_of_sale/static/tests/tours/receipt_screen_tour.js

```diff
@@ -54,7 +54,11 @@ registry.category("web_tour.tours").add("ReceiptScreenTour", {
             ProductScreen.addOrderline("Desk Pad", "6", "5"),
             ProductScreen.clickPayButton(),
             PaymentScreen.clickTipButton(),
-            NumberPopup.enterValue("1"),
```

Comment:
Why not `Numpad.click("1")` ?
It's true that after  f96fc3cb7e9070ae9a65724541df993d901e0ddc we can just remove `NumberPopup.enterValue`